<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchAddTemplateDlg</class>
 <widget class="QDialog" name="BatchAddTemplateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(180,180,180);
border-radius:5px </string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180);
border-top-right-radius:5px;
border-top-left-radius:5px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;黑体&quot;;
border:none</string>
    </property>
    <property name="text">
     <string>批量添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btnClose">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>0</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:5px;
border-top-left-radius:0px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>温馨提示：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>若首次上传可下载模板填写后上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnDownLoadTemplate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;黑体&quot;;
border: none;
background-color:rgb(180,180,180);
border-top-right-radius:2px;
border-top-left-radius:2px;
border-bottom-right-radius:2px;
border-bottom-left-radius:2px;</string>
   </property>
   <property name="text">
    <string>点击下载模板</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editFile">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>95</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:2px;
font: 18px &quot;等线&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnSelectFile">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;
border: none;
border-top-right-radius:2px;
border-top-left-radius:2px;
border-bottom-right-radius:2px;
border-bottom-left-radius:2px;
background-color:rgb(180,180,180)</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>*请按照模板样式上传文件，最大999M，支持xls等格式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnUpLoadFile">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
border: none;
border-radius: 2px;
background-color:rgb(180,180,180)</string>
   </property>
   <property name="text">
    <string>上传文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
border: none;
border-radius: 2px;
background-color:rgb(180,180,180)</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
